scripts/jenkins/ardana/ansible/roles/input_model_generator/vars/templates/service/full-spread.yml

```diff
@@ -89,7 +89,7 @@ service_groups:
     type: resource
     prefix: rhel-comp
     distro_id: rhel73-x86_64
-    heat_image_id: centos73
+    heat_image_id: centos73-ardana
```

Comment:
As a followup to #2901, could you please remove all `heat_image_id` explicit configurations from all service templates ? We should rely on the default settings and let the heat generator select the correct image based on the `distro-id`. This was left here just to exemplify that this is supported too.

Reply:
yes, that's great. Thanks!